src/data_structures/Translocation.cpp

```diff
@@ -123,6 +124,39 @@ void Window::insertRead(BamAlignment alignment) {
 
 }
 
+float Window::computeCoverageB(string bamFileName, int chrB, int start, int end, int32_t secondWindowLength){
+	int bases=0;
+	float coverage;
+	BamReader bamFile;
+	BamAlignment currentRead;
+
+
+	//test if an index file is available
+
```

Comment:
More than here I would check this at the beginnign of the program. If the bai  is not available you do not start the program at all